forks/jdt/pom.xml

```diff
@@ -19,7 +19,7 @@
             <groupId>org.eclipse.tycho</groupId>
             <artifactId>org.eclipse.jdt.core</artifactId>
             <version>3.10.0.v20140604-1726</version>
-            <scope>provided</scope>
+<!--             <scope>provided</scope> -->
```

Comment:
+1, I had hard time to have it in provided scope for Eclipse development and Junit test running. So I have commented that line temporary.